tv_apps/smart-system/js/app_modal_dialog.js

```diff
@@ -246,10 +247,15 @@
 
         if (evt.yesText) {
           option.buttonSettings[0].textRaw = evt.yesText;
+          textRawPromises.push(Promise.resolve());
```

Comment:
You don't need to add a `Promise.resolve()` here since `Promise.all([])` will be resolved as well.